decko-rails/lib/decko/engine.rb

```diff
@@ -27,7 +26,7 @@ def gem_root
 
   end
 
-  class Decko::Engine < Rails::Engine
+  class Engine < ::Rails::Engine
     
     paths.add "app/controllers", :eager_load => true
     paths.add 'gem-assets',      :with => 'public/assets'
```

Comment:
I'm not sure about that.  Not sure I totally understand how it should work either.

What inherits from what?  I never got clear on whether an Application and the Engines in may mount actually share config and paths.  Presumably if the can/should, I haven't configured it right with Railties, and if not, then I don't know what this comment implies.  I don't think Rails::Engine will have those in inherit, only instances of subclasses would (at least I think that is the case).